react/src/layout/form.tsx

```diff
@@ -79,17 +79,20 @@ export class FieldGridLayout extends Layout<FieldGridConfig, {}> {
         let schema = useContext(CSchema);
         let store = useStore();
 
+        if (! useShown(this.config.shown, true, formState, ValueSelectors.Fields)) {
```

Comment:
I am concerned with using hooks inside conditionals. Especially as we have new people working on React. This works because there are no other elements of the conditional but things like short-circuiting will cause an error. I think for hooks specifically it might be better to have them on their own lines

Reply:
Yes, that makes sense - I've changed this with b239f85